### PR TITLE
30041 update to new service

### PR DESCRIPTION
**Pull request by syre**

### CODE REVIEW

File: service_person_stamdata_udvidet/udvidet_person_stam_data_lokal.py [hunk 1]
```diff
@@ -17,13 +17,10 @@
...
-service_url = (
-        "https://prod.serviceplatformen.dk/service/CPRInformation/"
-        "CPRInformation/1"
-    )
+service_url = "https://prod.serviceplatformen.dk/service/CPR/PersonBaseDataExtended/4"
 
 
-def get_citizen(service_uuids, certificate, cprnr):
+def get_citizen(service_uuids, certificate, cprnr, service_url=service_url):
     """The function returnes a citizen dict from the
```
Comment (istherepie) [inline service_person_stamdata_udvidet/udvidet_person_stam_data_lokal.py:29]: We may want to add both the URI for production and test to the codebase and change the parameter to a toggle between them, e.g. 

`get_citizen(service_uuids, certificate, cprnr, production=False):`

**Review by istherepie — CHANGES_REQUESTED** [review]
Looks good!
Let's discuss whether the proposed change is necessary.

File: service_person_stamdata_udvidet/udvidet_person_stam_data_lokal.py (hunk 1 above)
Comment (syre) [reply service_person_stamdata_udvidet/udvidet_person_stam_data_lokal.py:29]: :+1:

**Review by istherepie — APPROVED** [review]

**Pull request merged**